src/code/irqmgr.c

```diff
@@ -127,54 +179,64 @@ void IrqMgr_CheckStack() {
 }
 
 void IrqMgr_HandlePRENMI450(IrqMgr* this) {
-    u64 temp = STATUS_NMI; // required to match
-    gIrqMgrResetStatus = temp;
-    this->resetStatus = STATUS_NMI;
+    u64 nmi = IRQ_RESET_NMI; // required to match
+
+    gIrqMgrResetStatus = nmi;
+    this->resetStatus = IRQ_RESET_NMI;
 
-    osSetTimer(&this->timer, OS_USEC_TO_CYCLES(30000), 0ull, &this->queue, (OSMesg)PRENMI480_MSG);
-    IrqMgr_SendMesgForClient(this, (OSMesg) & this->nmiMsg);
+    // Schedule a PRENMI480 message to be handled in 30ms
+    osSetTimer(&this->timer, OS_MSEC_TO_CYCLES(30), 0, &this->queue, (OSMesg)PRENMI480_MSG);
+    // Send the NMI event to clients
+    IrqMgr_SendMesgToClients(this, (OSMesg) & this->nmiMsg);
 }
 
 void IrqMgr_HandlePRENMI480(IrqMgr* this) {
     u32 ret;
 
-    osSetTimer(&this->timer, OS_USEC_TO_CYCLES(20000), 0ull, &this->queue, (OSMesg)PRENMI500_MSG);
+    // Schedule a PRENMI500 message to be handled in 20ms
+    osSetTimer(&this->timer, OS_MSEC_TO_CYCLES(20), 0, &this->queue, (OSMesg)PRENMI500_MSG);
+
     ret = osAfterPreNMI();
-    if (ret) {
+    if (ret != 0) {
```

Comment:
`ret` doesn't look like a very good name for this since you'd normally expect it to be returned by this function. Would `preNMIComplete` be suitable?